1-Basic/README.md

```diff
@@ -115,25 +116,21 @@ where `[APPLICATIONID]` is the identifier you copied in step 2. Save the file.
 
 #### [OPTIONAL] Step 3b: Configure the Android project with your return URI
 
-1. Open the `UserDetailsClient.Droid\MainActivity.cs` file.
-2. Open the `UserDetailsClient.Droid\Properties\AndroidManifest.xml`
-3. Add or modify the `<application>` element as in the following:
-
-```Xml
-    <application>
-    <activity android:name="microsoft.identity.client.BrowserTabActivity">
-      <intent-filter>
-    <action android:name="android.intent.action.VIEW" />
-    <category android:name="android.intent.category.DEFAULT" />
-    <category android:name="android.intent.category.BROWSABLE" />
-    <data android:scheme="msal[APPLICATIONID]" android:host="auth" />
-      </intent-filter>
-    </activity>
-      </application>
+1. Open the `UserDetailsClient.Droid\MsalActivity.cs` file.
+1. Replace `[APPLICATIONID]` with the identifier you copied in step 2.
+1. Save the file.
+
+```csharp
+  [Activity]
+  [IntentFilter(new[] { Intent.ActionView },
+        Categories = new[] { Intent.CategoryBrowsable, Intent.CategoryDefault },
+        DataHost = "auth",
+        DataScheme = "msal[APPLICATIONID]")]
```

Comment:
nit: can this be client id? or change client id to application id? it's always confusing to have both.

Reply:
Renamed